### PR TITLE
[dwpost] Cleanups and type-hint rewrite

### PR DESCRIPTION
**Pull request by Quix0r**
- used type-hint `App` and no call-by-reference
- converted spaces -> tab for indenting
- DBA::isResult() used instead of empty/count/...
- added curly braces and spaces for better readability
- opening curly brace after function/method/class belongs into new line
- converted multiple single-line comments to single multi-line comment

### CODE REVIEW

File: dwpost/dwpost.php [hunk 1]
```diff
@@ -8,177 +8,184 @@
...
 
-	// dreamwidth post in the LJ user's timezone.
-	// Hopefully the person's Friendica account
-	// will be set to the same thing.
+	if ($b['parent'] != $b['id']) {
+		return;
+	}
 
+	/*
+	 * dreamwidth post in the LJ user's timezone.
```
Comment (tobiasd) [inline dwpost/dwpost.php]: I think this LJ should be a DW in the comment.

Comment (annando) [reply dwpost/dwpost.php]: What have I overseen?

Comment (tobiasd) [reply dwpost/dwpost.php]: Nothing serious. A leftover from the creation of the addon in the comments I think. Looks liky Tony made the LJ addon and then based the DW addon on it and that LJ in the comment above is a hint to that.

Comment (annando) [reply dwpost/dwpost.php]: Ah okay.

**Pull request merged**